index.js

```diff
@@ -99,6 +99,9 @@ function doInject (dispatchFunc, options, callback) {
 }
 
 function Chain (dispatch, option) {
+  if (typeof option === 'string') {
+    option = { url: option }
+  }
```

Comment:
Can we just assign `this.option` here and wrap the next line in a `else` block? Because if `option` is a string, there is no need to call `Object.assign`.

Reply:
With `object.assign` you create a shallow copy of the options to prevent side effects that's fine.